src/internal/operations/backup.go

```diff
@@ -424,11 +425,20 @@ func (op *BackupOperation) do(
 		lastBackupVersion = mans.MinBackupVersion()
 	}
 
+	// Use sliding window limiter for Exchange if the feature is not explicitly
+	// disabled. For other services we don't use sliding window limiter.
+	enableSlidingLim := false
+	if service == path.ExchangeService &&
+		!op.Options.ToggleFeatures.DisableSlidingWindowLimiter {
+		enableSlidingLim = true
+	}
+
```

Comment:
We're getting into bad habits here.  The operation layer should not investigate the service being handled.  These sorts of conditionals need to be handled via an interface that's common to all backup providers.  Something like `ctx = op.backupProducer.SetRateLimiter(ctx, op.Selector, op.Options)`.

Reply:
Good point! I'll make the change.